lisa/tools/lscpu.py

```diff
@@ -43,9 +57,9 @@ class Lscpu(Tool):
     __valid_architecture_list = ["x86_64"]
     # Capture patterns of the form `0 0 0 0:0:0:0`
     _core_numa_mappings = re.compile(
-        r".*(?P<cpu>\d+).+(?P<numa_node>\d+).+(?P<socket>\d+)"
-        r".+(?P<l1_data_cache>\d+):(?P<l1_instruction_cache>\d+)"
-        r":(?P<l2_cache>\d+):(?P<l3_cache>\d+)$"
+        r"\s*(?P<cpu>\d+)\s+(?P<numa_node>\d+)\s+(?P<socket>\d+)\s+"
```

Comment:
Add newly found examples above. So if you find more instances in future, they can be tested together.

Reply:
It's not a new example as such. Earlier regex wasn't capturing two digit `cpu_id`. The intent that CPU id could be a n digit number should be clear from the current comment.

Reply:
can it capture the example, "0 0 0 0:0:0:0" before? If so, what's improved by changing this pattern?

Reply:
`0 0 0:0:0:0` is an example pattern where every 0 signifies a integer. How about I can update it to `[int] [int] [int]:[int]:[int]:[int]` to be explicit about n-digit integer?

Reply:
It's better to have real cases. Once it needs to include more cases, people can easily do regression testing from those cases. Check examples in operating_system.py. The patterns are updated by new findings, the examples can help preventing regression.